manager/pom.xml

```diff
@@ -157,6 +222,15 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>17</source>
+                    <target>17</target>
+                    <compilerArgs>--enable-preview</compilerArgs>
```

Comment:
Any reason we are enabling preview?

Reply:
No this is a leftover, I've pushed a fix: https://github.com/gingersnap-project/cache-manager/commit/1775af995f3f6d7d0ea6023cfc55a1c8c787e605